addons/cloudfront_distribution.tf

```diff
@@ -194,7 +221,7 @@ resource "aws_cloudfront_distribution" "cloudfront" {
   }
 
   logging_config {
-    bucket = var.cf_log_bucket
+    bucket = "${var.cf_log_bucket}.s3.amazonaws.com"
```

Comment:
can we make this a local that only adds .s3.amazonaws.com if it's not already present in the var.cf_log_bucket variable? This will help us with backwards compatability.

Reply:
Just to explain, cloudfront logging configuration requires an extension `.s3.amazonaws.com` regardless how you create a bucket, i.e. `cloudfront-logs.s3.amazonaws.com` or `clodfront-logs.s3.amazonaws.com.s3.amazonaws.com`. So I recon create a bucket without extension and  let template take care of it.

Reply:
Issue is we, and others are using pre-existing buckets `cloudfront-logs` and using 

```
cf_log_bucket_create = false
cloudfront_log_bucket = cloudfront_log_bucket.s3.amazonaws.com
```

by making this change backwards compatible we can support these users until 1.8 release when we can remove backwards compatible fixes like this.

Reply:
I tested this scenario too and we are still supporting existing functionality - bucket creation is optional. As I mentioned, if you configure this -
```
cf_log_bucket_create = false
cloudfront_log_bucket = cloudfront_log_bucket.s3.amazonaws.com
```
make sure your bucket has correct configuration, ACL and policy.

Reply:
And worth to point out that our pre-existing bucket for cf log are not correctly configured so that's why we not receiving any logs